workflow/energy_rating_index.rb

```diff
@@ -717,35 +719,59 @@ def write_eri_results(results, resultsdir, design_outputs, results_iad, csv_file
   end
 end
 
-def write_es_results(resultsdir, esrd_eri_results, rated_eri_results, rated_eri_results_wo_opp, target_eri, saf, passes)
-  esrd_eri = esrd_eri_results[:eri].round(0)
+def write_es_zerh_results(ruleset, resultsdir, rd_eri_results, rated_eri_results, rated_eri_results_wo_opp, target_eri, saf, passes)
+  rd_eri = rd_eri_results[:eri].round(0)
   target_eri = target_eri.round(0)
   rated_eri = rated_eri_results[:eri].round(0)
   rated_wo_opp_eri = rated_eri_results_wo_opp[:eri].round(0)
 
-  if rated_wo_opp_eri - rated_eri > esrd_eri - target_eri
+  if rated_wo_opp_eri - rated_eri > rd_eri - target_eri
     fail 'Unexpected error.'
   end
 
-  results_csv = File.join(resultsdir, 'ES_Results.csv')
-  results_out = []
-  results_out << ['Reference Home ERI', esrd_eri]
-  if saf.nil?
-    results_out << ['SAF (Size Adjustment Factor)', 'N/A']
-  else
-    results_out << ['SAF (Size Adjustment Factor)', saf.round(3)]
-  end
-  results_out << ['SAF Adjusted ERI Target', target_eri]
-  results_out << [nil] # line break
-  results_out << ['Rated Home ERI', rated_eri]
-  results_out << ['Rated Home ERI w/o OPP', rated_wo_opp_eri]
-  results_out << [nil] # line break
-  if passes
-    results_out << ['ENERGY STAR Certification', 'PASS']
-  else
-    results_out << ['ENERGY STAR Certification', 'FAIL']
+  if ESConstants.AllVersions.include? ruleset
```

Comment:
Too much duplication of code here. I suggest handling the differences up front, e.g.:
```
if ESConstants.AllVersions.include? ruleset
  program_abbreviation, program_name = 'ES', 'ENERGY STAR'
elsif ZERHConstants.AllVersions.include? ruleset
  program_abbreviation, program_name = 'ZERH', 'Zero Energy Ready Home'
end
```
and then reference these variables in the code.